java/src/jmri/jmrit/logixng/tools/swing/EditErrorHandlingModuleAction.java

```diff
@@ -19,7 +19,15 @@ public class EditErrorHandlingModuleAction extends AbstractAction {
     private ErrorModuleEditor _errorModuleEditor;
 
     public EditErrorHandlingModuleAction() {
-        super(Bundle.getMessage("TitleErrorHandlingModuleEditor"));
+        super(getTitle());
```

Comment:
The title is only set during JMRI startup.  Subsequent changes, such as loading or creating the module do not change the title.

Reply:
The last change handles changes to the module state but does not update the menu label after loading the xml data file if it has the error module.

Reply:
Thanks. Yes, I know. I'm working on it.

The problem is that it's not enough to listen on the manager if the module is created. I need to listen on when LogixNG is initialized and then check if the module exists and is not empty. So I need to add a property change event to the initialization of LogixNG.

Reply:
Could you add an initial check in `logixNG_Manager.setupAllLogixNGs();`.